chat-widget/src/components/livechatwidget/common/initWebChatComposer.ts

```diff
@@ -66,6 +66,10 @@ export const initWebChatComposer = (props: ILiveChatWidgetProps, state: ILiveCha
                 dispatch({ type: LiveChatWidgetActionType.SET_CONVERSATION_ENDED_BY, payload: ConversationEndEntity.Agent });
             }
 
+            if (conversationDetails?.participantType === ParticipantType.Bot || conversationDetails?.participantType === ParticipantType.User) {
+                dispatch({ type: LiveChatWidgetActionType.SET_POST_CHAT_PARTICIPANT_TYPE, payload: conversationDetails?.participantType });
+            }
+
```

Comment:
duplicated code, and same comment as in end chat, there are only 2 types of participants, no need to check on those types, is it?

Reply:
Same explanation as above